### ISSUE TITLE
require 'config/config.php' w `includes`

### ISSUE THREAD

**Issue by websafe** [LOCOSP/interface_web_home_automation#14]
W plikach w `includes/` unikaj kolejnych `require`, czy `include`, bo stracisz panowanie nad tym. 
Na początek wywal z `header.php`:

~~~~
<?php $cfg = require 'config/config.php'; ?>
~~~~

i wstaw do plików w głównym katalogu, na samym początku:

~~~~
<?php require 'config/config.php'; ?>
~~~~

a sam plik `config/config/php` zmień delikatnie na:

~~~~
$cfg = array(
    'dreambox' => array(
        'ip' => '192.168.1.128',
        'port' => 80,
    )
);
~~~~

**Comment by websafe** [comment]
@LOCOSP mały update w moim komentarzu zrobiłem kiedy Ty jużedytowałeś header.

**Comment by LOCOSP** [comment]
Teraz zabardzo nie wiem w jakie pliki z katalogu głównego powstawiać: 

<?php require 'config/config.php'; ?>

**Comment by websafe** [comment]
tylko `index.php`, rzuć okiem na #15

**Comment by websafe** [comment]
TYlko `index.php` ale ten nowy-pusty (patrz#15). W tym, który przemianujesz na `home.php` nie dawaj tego require.

**Comment by LOCOSP** [comment]
ok #15 już widziałem, zrobię to wieczorkiem ;)

**Comment by websafe** [comment]
@LOCOSP tak, ale chodziło mi o to z index.php, ze w #15 jest opisana zmiana samego `index.php` i #14 powinieneś robić rozumiejąc co zaraz zrobisz w #15 :-)
